AndyDorokhin/HW6/task_2.py

```diff
@@ -0,0 +1,17 @@
+""" 
```

Comment:
good job
look at this code:

def greet_first():
    while True:
        login = input("Enter login: ")
        if login.lower() == "first":
            break
        print("Wrong login. Please enter login again.")

    print(f"Hello, {login.capitalize()}!")

def main():
    greet_first()

if __name__ == "__main__":
    main()